app/lib/google_extensions/attached_disk.rb

```diff
@@ -0,0 +1,13 @@
+module GoogleExtensions
+  module AttachedDisk
+    def persisted?
+      type == 'PERSISTENT'
+    end
```

Comment:
Are you sure about this? I'd assume this is `persisted?` as rails understand it (https://apidock.com/rails/v5.2.3/ActiveRecord/Persistence/persisted%3F), not how we or google intuitively understand it

Reply:
Good point, I'll leave it empty then